src/commons/sideContent/githubAssessments/SideContentMissionEditor.tsx

```diff
@@ -0,0 +1,108 @@
+import { InputGroup, Label } from '@blueprintjs/core';
+import { DatePicker } from '@blueprintjs/datetime';
+import { Variant } from 'js-slang/dist/types';
+import React from 'react';
+
+import { SourceLanguage } from '../../application/ApplicationTypes';
+import { ControlBarChapterSelect } from '../../controlBar/ControlBarChapterSelect';
+import { MissionMetadata } from '../../githubAssessments/GitHubMissionTypes';
+import Constants from '../../utils/Constants';
+
+export type SideContentMissionEditorProps = {
+  missionMetadata: MissionMetadata;
+  setMissionMetadata: (missionMetadata: MissionMetadata) => void;
+};
+
+const SideContentMissionEditor: React.FC<SideContentMissionEditorProps> = props => {
+  const datePicker =
+    props.missionMetadata.dueDate.getFullYear() > new Date().getFullYear() ? (
+      <DatePicker onChange={handleDateChange} />
+    ) : (
+      <DatePicker onChange={handleDateChange} defaultValue={props.missionMetadata.dueDate} />
+    );
```

Comment:
the calendar UI seems a little off 
![image](https://user-images.githubusercontent.com/53928333/121198678-e9584500-c8a4-11eb-8e92-da7c658e646a.png)